tests/eval-one.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
Evaluating just one is not a proper test, because profiles can be inherited. It can be done via `nixos-rebuild -I nixos-config=tests/eval-test.nix nixos-hardware-config=path/to/default.nix dry-build`. I think it's a bad idea to allow to eval just one via a script.

Reply:
Travis will evaluate all of them in the end, or not?

Reply:
Travis evaluates every profile on each commit/PR.

Reply:
The idea here is that I can test my profile without having to wait on all the others to evaluate. Travis is testing all the profiles anyways.

Reply:
You shouldn't assume that your profile isn't inherited by any other profiles. Profile might be inherited by someone from another directory, e.g. in case of brand renaming (e.g. IBM -> Lenovo).

Reply:
How is this relevant? Travis will catch this eventually.

Reply:
It's relevant in that it's not a test and thus should not be exposed. It's possible to see someone with merge rights using `eval-one` test and then pushing to `master`.